docs/developer-guide/nodejs-api.md

```diff
@@ -30,8 +30,6 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
     * [getRules()](#lintergetrules)
     * [defineParser()](#linterdefineparser)
     * [version](#linterversionlinterversion)
-* [linter (deprecated)](#linter-1)
-* [CLIEngine (deprecated)](#cliengine)
 * [RuleTester](#ruletester)
     * [Customizing RuleTester](#customizing-ruletester)
 * [Deprecated APIs](#deprecated-apis)
```

Comment:
"Deprecated APIs" section is also removed

Reply:
```suggestion
```

Reply:
Thanks!